Kitodo-API/src/main/java/org/kitodo/api/Metadata.java

```diff
@@ -11,11 +11,28 @@
 
 package org.kitodo.api;
 
+import org.kitodo.api.dataformat.mets.MdSec;
+
 public class Metadata {
+    /**
+     * In which conceptual area in the METS file this meta-data entry is stored.
+     */
+    private MdSec domain;
 
     // The key of the metadata.
     private String key;
 
+    /**
+     * Specifies the location of the meta-data entry in the METS file. METS
+     * allows the storage of meta-data in five different areas with different
+     * semantics. See {@link MdSec} for details.
+     *
+     * @return the location of the meta-data entry
+     */
+    public MdSec getDomain() {
```

Comment:
please do not reference a class in a lower api package.
I think you need to move te enum next to the Metadata class